internal/cmd/commands/credentialstorescmd/vault_funcs.go

```diff
@@ -0,0 +1,186 @@
+package credentialstorescmd
+
+import (
+	"github.com/hashicorp/boundary/api/credentialstores"
+	"github.com/hashicorp/boundary/internal/cmd/base"
+)
+
+func init() {
+	extraVaultFlagsFunc = extraVaultFlagsFuncImpl
+	extraVaultActionsFlagsMapFunc = extraVaultActionsFlagsMapFuncImpl
+	extraVaultFlagsHandlingFunc = extraVaultFlagHandlingFuncImpl
+}
+
+const (
+	addressFlagName              = "vault-address"
+	namespaceFlagName            = "vault-namespace"
+	vaultCaCertFlagName          = "vault-ca-cert"
+	tlsServerNameFlagName        = "vault-tls-server-name"
+	tlsSkipVerifyFlagName        = "vault-tls-skip-verify"
+	vaultTokenFlagName           = "vault-token"
+	clientCertificateFlagName    = "vault-client-certificate"
+	clientCertificateKeyFlagName = "vault-client-certificate-key"
+)
+
+type extraVaultCmdVars struct {
+	flagAddress       string
+	flagNamespace     string
+	flagCaCert        string
+	flagVaultToken    string
+	flagClientCert    string
+	flagClientCertKey string
+	flagTlsServerName string
+	flagTlsSkipVerify bool
+}
+
+func extraVaultActionsFlagsMapFuncImpl() map[string][]string {
+	flags := map[string][]string{
+		"create": {
+			addressFlagName,
+			namespaceFlagName,
+			vaultCaCertFlagName,
+			tlsServerNameFlagName,
+			tlsSkipVerifyFlagName,
+			vaultTokenFlagName,
+			clientCertificateFlagName,
+			clientCertificateKeyFlagName,
+		},
+	}
+	flags["update"] = flags["create"]
+	return flags
+}
+
+func extraVaultFlagsFuncImpl(c *VaultCommand, set *base.FlagSets, _ *base.FlagSet) {
+	f := set.NewFlagSet("Vault Credential Store Options")
+
+	for _, name := range flagsVaultMap[c.Func] {
+		switch name {
+		case addressFlagName:
+			f.StringVar(&base.StringVar{
+				Name:   addressFlagName,
+				Target: &c.flagAddress,
+				Usage:  "The vault address.",
```

Comment:
```suggestion
				Usage:  "The address of the Vault server. This should be a complete URL such as https://127.0.0.1:8200",
```
Taken from the Vault Agent [configuration page](https://www.vaultproject.io/docs/agent#address).